[2025-03-06T16:35:46.553+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-06T16:35:46.576+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_setup_dag.transform_write_data manual__2025-03-06T16:35:40.387223+00:00 [queued]>
[2025-03-06T16:35:46.586+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_setup_dag.transform_write_data manual__2025-03-06T16:35:40.387223+00:00 [queued]>
[2025-03-06T16:35:46.586+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-03-06T16:35:46.608+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): transform_write_data> on 2025-03-06 16:35:40.387223+00:00
[2025-03-06T16:35:46.617+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=81) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-06T16:35:46.618+0000] {standard_task_runner.py:72} INFO - Started process 90 to run task
[2025-03-06T16:35:46.622+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_setup_dag', 'transform_write_data', 'manual__2025-03-06T16:35:40.387223+00:00', '--job-id', '53', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/tmp/tmpo_9y1bf0']
[2025-03-06T16:35:46.624+0000] {standard_task_runner.py:105} INFO - Job 53: Subtask transform_write_data
[2025-03-06T16:35:47.053+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_setup_dag.transform_write_data manual__2025-03-06T16:35:40.387223+00:00 [running]> on host 255716c1882e
[2025-03-06T16:35:47.261+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_setup_dag' AIRFLOW_CTX_TASK_ID='transform_write_data' AIRFLOW_CTX_EXECUTION_DATE='2025-03-06T16:35:40.387223+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-06T16:35:40.387223+00:00'
[2025-03-06T16:35:47.273+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-06T16:35:47.385+0000] {logging_mixin.py:190} INFO - Successfully loaded raw weather data from /opt/***/dags/data/raw_weather_data.json
[2025-03-06T16:35:47.454+0000] {logging_mixin.py:190} INFO - Successfully transformed weather data into DataFrame
[2025-03-06T16:35:47.479+0000] {logging_mixin.py:190} INFO - Data saved to /opt/***/dags/data/clean_weather_data.csv
[2025-03-06T16:35:47.480+0000] {logging_mixin.py:190} INFO - Data successfully saved to /opt/***/dags/data/clean_weather_data.csv
[2025-03-06T16:35:47.480+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-06T16:35:47.498+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-06T16:35:47.499+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=etl_setup_dag, task_id=transform_write_data, run_id=manual__2025-03-06T16:35:40.387223+00:00, execution_date=20250306T163540, start_date=20250306T163546, end_date=20250306T163547
[2025-03-06T16:35:47.591+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-06T16:35:48.137+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-06T16:35:48.142+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
